openshift/ftest/src/test/resources/arquillian.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
 	<extension qualifier="openshift">
-		<property name="definitionsFile">src/test/resources/hello_pod.json</property>
+		<property name="env.config.url">src/test/resources/hello_pod.json</property>
```

Comment:
@lordofthejars: I'd like @dipak-pawar to confirm first, in case I miss something. 

I tried to address #626 with #627. If that makes sense, then we could close this one.